lib/Core/Executor.h

```diff
@@ -147,6 +147,11 @@ class Executor : public Interpreter {
   /// \invariant \ref addedStates and \ref removedStates are disjoint.
   std::vector<ExecutionState *> removedStates;
 
+  /// Used to track states that are only removed from the searcher
+  std::vector<ExecutionState *> pausedStates;
+  /// Used to track states that are only added to the searcher
```

Comment:
Can you expand on this comment? What does "only added to the searcher" means, exactly?